utxo-info/README.md

```diff
@@ -0,0 +1,19 @@
+```
+shortname: ?/UTXO-IMPL
+name: A short discription of the current UTXO implementation
+type: informational
+status: Draft
+editor: Vanshdeep Singh <vanshdeep@bigchaindb.com>
+```
+
+## Description
+The UTXO (unspent transaction output) of each transaction is tracked using the a merkle tree. The merkle root of this tree is used as the app hash and retuned to Tendermint during `commit`. Below is a short summary of the UTXO is calculated,
```

Comment:
Maybe link to some articles about Merkle trees.

https://en.wikipedia.org/wiki/Merkle_tree

What about AVL trees, IAVL trees, and IAVL+ trees. I gather that those are supposed to be useful somehow, as an improvement. Is it the rules for adding the next leaf UTXO, where to put it?

Reply:
@ttmc I have restrained the context of this BEP to just discussing the current implementation. Also, the leaves of the UTXO merkle tree are added to the list of the existing leaves and the list is sorted after which merkle root is calculated. As explained in the BEP, the leaves are added to the `utxo` collection and then all the leaves are read back and sorted, after which the root is calculated.